pages/instructorAssessmentAccess/instructorAssessmentAccess.js

```diff
@@ -10,18 +11,164 @@ const sqlLoader = require('../../prairielib/lib/sql-loader');
 
 const sql = sqlLoader.loadSqlEquiv(__filename);
 
-router.get('/', function (req, res, next) {
-  debug('GET /');
-  var params = {
-    assessment_id: res.locals.assessment.id,
-    link_exam_id: config.syncExamIdAccessRules,
-  };
-  sqldb.query(sql.assessment_access_rules, params, function (err, result) {
-    if (ERR(err, next)) return;
-    res.locals.access_rules = result.rows;
+let compare_date = function (old_date, new_date, old_is_null, new_is_null, both_are_null) {
```

Comment:
Nit: we apply this rule inconsistently (and not at all for data coming from database queries), but for your consideration, we typically use `camelCase` for function names and parameters.